src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -447,7 +447,18 @@ internal PSModuleInfo LoadUsingMultiVersionModuleBase(string moduleBase, Manifes
         /// <summary>
         /// Extra variables that are allowed to be referenced in module manifest file.
         /// </summary>
-        private static readonly string[] s_extraAllowedVariables = new string[] { SpecialVariables.PSScriptRoot, SpecialVariables.PSEdition, SpecialVariables.EnabledExperimentalFeatures };
+        private static readonly string[] s_extraAllowedVariables = new string[] {
+            SpecialVariables.PSScriptRoot,
+            SpecialVariables.PSHome,
+            SpecialVariables.PSEdition,
+            SpecialVariables.PSCulture,
+            SpecialVariables.PSUICulture,
+            SpecialVariables.IsCoreCLR,
+            SpecialVariables.IsLinux,
+            SpecialVariables.IsMacOS,
+            SpecialVariables.IsWindows,
+            SpecialVariables.EnabledExperimentalFeatures
+        };
```

Comment:
Suggestion: it would be easier at a glance to see what is here if these were in alphabetical order.

Also, a question: why `$PSHome`? The rest I get, but I don't understand why that one is included.

Reply:
I think for `$Home` as well as for any new cmdlet that is allowed, it would be helpful for me to see specific (not contrived) examples that show why a manifest is better with them.

Reply:
@KirkMunro I post a report on the main page of the PR. If you are still again Home/PSHome, i will remove them. But keep in mind that env:* is allowed, for me, adding them it's more about consistency.

Reply:
@fMichaleczek You're right, since `$env:` is allowed, that makes sense.